tpls/runtime-template.js

```diff
@@ -204,5 +204,22 @@ function applyUpdate(callback, errback) {
   <% } %>
 }
 
+function updateOfflineService() {
+  if (navigator.serviceWorker) {
+    navigator.serviceWorker.ready.then((registration) => {
+      if(!registration) {
+        return;
+      }
+      return registration.update();
+    });
+  } else if (window.applicationCache) {
+    applicationCache.update();
+  }
+}
+
+<% if (typeof autoUpdate !== 'undefined' && autoUpdate === true) { %>
+  setInterval(function () { updateOfflineService(); }, <%- JSON.stringify(autoUpdateInterval) %>);
```

Comment:
`<%- autoUpdateInterval %>` should be fine I guess (will generated number instead of string).